src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/ITelemetrySession.cs

```diff
@@ -1,6 +1,8 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using NuGet.Common;
```

Comment:
do you really need to import this namespace here?

Reply:
Yes, because telemetryEvent is in NuGet.Common